### ISSUE TITLE
Unable to tab past Suggestion dropdown

### ISSUE THREAD

**Issue by unekinn** [digdir/designsystemet#3414]
### Version number

1.0.2

### Description of the bug

When the Suggestion input component gains focus, it opens its dropdown. While the dropdown is open, pressing `tab` does not move focus away from the dropdown. You can press `esc` to close the dropdown, but pressing `tab` opens it again

### Steps To Reproduce

1. Go to https://storybook.designsystemet.no/?path=/docs/komponenter-suggestion--docs
2. Click "Velg en destinasjon", which opens the dropdown
3. Press `tab`, notice that the focus does not move

**Comment by Barsnes** [comment]
Adding `tabindex="-1"` to `u-datalist` fixes this problem

**Comment by eirikbacker** [comment]
Hi @unekinn ! What browser is this in? Is the error possible to reproduce for you at https://u-elements.github.io/u-elements/elements/u-datalist? ☺

**Comment by eirikbacker** [comment]
Ah! Figured it out - `overflow: auto` actually [causes focusability](https://cassey.dev/til/2019-11-19-overflow-scroll-gets-focus/). `u-datalist@1.0.5` is released with `tabindex="-1"` included automatically <3
